docs/dapp/sapphire/gsn.md

```diff
@@ -0,0 +1,76 @@
+---
+description: CLI instructions for deploying GSN on Sapphire Testnet
+---
+# Gas Station Network
+
+[Gas Station Network](https://docs.opengsn.org) is very useful for improving user experiences of many Dapps by making their users send transactions without having native tokens for gas. We make GSN functionalities available on Sapphire Paratime.
+
+### Package Install
+
+Starting with an empty folder, let us install the GSN command line tool by using the following commands:
```

Comment:
```suggestion
Starting with an empty folder, let us install the [Oasis fork] of the GSN command line tool by using the following commands:
```

Can you also cleanup and merge https://github.com/oasislabs/gsn/pull/1 so we have a fork publicly ready? We do need a fork and upstream GSN doesn't work with Sapphire right?